### DISCUSSION TITLE
I am getting charged on github for unknown reasons

### DISCUSSION THREAD

**Discussion by Mutahharahmad97** [community/community#50575] [Copilot Conversations]
### Select Topic Area

General

### Body

I took GitHub copilot yearly subscription and i got two months free. I have to pay $100 which i haven't paid yet but i am getting charged on daily basis. Right now my bill is $183 and i am very confused as the usage shows $0 used this month. Kindly help me understand this. 

Thank you!

**Comment by Akash1134** [comment]
Hi @Mutahharahmad97 Sorry about the trouble caused. I'd recommend getting in touch with our Support team for a thorough investigation by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
